### ISSUE TITLE
Opening PWA opens Brave

### ISSUE THREAD

**Issue by MarioMatschgi** [brave/brave-browser#16388]
## Description 

Installing a PWA, quitting Brave and then launching the PWA causes Brave to open as well with a new window

## Steps to Reproduce

   1. Go to a PWA ([My own PWA](https://recipe4you.programario.at), [Spotify](https://open.spotify.com/))
   2. Install the PWA (click "install")
   3. Close the PWA and Brave
   4. Open the PWA

## Actual result:

The PWA opens AND Brave opens as well

## Expected result:
The PWA opens BUT NOT Brave

## Reproduces how often: 

Easily reproduced, just open any installed PWA while Brave is quit

## Brave version (brave://version info)

`
Brave | 1.25.72 Chromium: 91.0.4472.101&nbsp;(Official Build)&nbsp;(x86_64)
-- | --
Revision | af52a90bf87030dd1523486a1cd3ae25c5d76c9b-refs/branch-heads/4472@{#1462}
OS | macOS Version 11.4 (Build 20F71)
JavaScript | V8&nbsp;9.1.269.36
User agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.101 Safari/537.36
Command Line | /Applications/Brave Browser.app/Contents/MacOS/Brave Browser --enable-dom-distiller --disable-domain-reliability --no-pings --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --variations-server-url=https://variations.brave.com/seed --enable-features=WebUIDarkMode,AutoupgradeMixedContent,SafetyTip,LegacyTLSEnforced,ReducedReferrerGranularity,PasswordImport,PrefetchPrivacyChanges --disable-features=NotificationTriggers,HandwritingRecognitionWebPlatformApi,FledgeInterestGroups,IdleDetection,FlocIdComputedEventLogging,HandwritingRecognitionWebPlatformApiFinch,InterestCohortFeaturePolicy,LiveCaption,AutofillServerCommunication,SubresourceWebBundles,NetworkTimeServiceQuerying,LangClientHintHeader,WebOTP,SignedExchangePrefetchCacheForNavigations,SharingQRCodeGenerator,TrustTokens,DirectSockets,InterestCohortAPIOriginTrial,FledgeInterestGroupAPI,AutofillEnableAccountWalletStorage,FirstPartySets,FederatedLearningOfCohorts,EnableProfilePickerOnStartup,TextFragmentAnchor,SignedExchangeSubresourcePrefetch,TabHoverCards --flag-switches-begin --flag-switches-end
Executable Path | /Applications/Brave Browser.app/Contents/MacOS/Brave Browser
Profile Path | /Users/Mario/Library/Application Support/BraveSoftware/Brave-Browser/Default
Variations | aed3cac0-3f4a17df7146a73c-3f4a17dfc63a3c82-3f4a17df
`

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Yes

- Can you reproduce this issue with the beta channel? 
- Yes

- Can you reproduce this issue with the nightly channel? 
- Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- No

- Does the issue resolve itself when disabling Brave Rewards? 
- No

- Is the issue reproducible on the latest version of Chrome? 
- Yes (Chrome 91.0.4472.101)

**Comment by aaaamirabbas** [comment]
+1

**Comment by jeannnemelanie** [comment]
Same issue here, with 2 differents PWA 😢
Any news for this ticket ?

**Comment by MarioMatschgi** [comment]
> Same issue here, with 2 differents PWA 😢
> Any news for this ticket ?

Sadly not - still same issue, even on the newest version of chrome
Also quitting the browser while a PWA is opened will also close the the app, even though they are separate "apps" (in macOS Task-switcher, TaskManager, doc, etc.

Seems to be an issue of chromium itself

**Comment by jeannnemelanie** [comment]
Yes I can confirm, same thing is happening with Google Chrome PWA. And quitting the browser while a PWA is opened will also close the the app in both Chrome & Brave. Hope Chromium will work on it.

**Comment by iamMG** [comment]
Still no solution for this?

**Comment by webknows** [comment]
Here's to hoping...

**Comment by rabinadk1** [comment]
I also can reproduce the issue.

**Comment by JStrategic** [comment]
Interesting that this still hasn't been addressed. Seems to be a long standing issue.

**Comment by AntoG87** [comment]
Same happens to me. Is there any update on this? Thanks

**Comment by s22-tech** [comment]
I'm experiencing the same problem using the latest version of Brave.  I'm kinda surprised this has been going on for so long.  Any movement on this issue?

**Comment by noahlschneider** [comment]
I am having a related issue. When I quit Brave, it closes my PWAs as well. I'd like to quit them separately. Is this related to this issue or should I file a separate issue? Thank you.
